### PR TITLE
Reduce the dependency on instance group in gke deployer

### PR DESCRIPTION
**Pull request by Argh4k**

### CODE REVIEW

File: kubetest/gke.go [hunk 1]
```diff
@@ -763,14 +774,9 @@ func (g *gkeDeployer) parseInstanceGroupsFromGcloud(igs string) ([]*ig, error) {
...
-	if err := g.getInstanceGroups(); err != nil {
-		return "", err
-	}
 	// We want to ensure that there's an e2e-ports-* firewall rule
-	// that maps to the cluster nodes, but the target tag for the
-	// nodes can be slow to get. Use the hash from the lexically first
-	// node pool instead.
-	return "e2e-ports-" + g.instanceGroups[0].uniq, nil
+	// that maps to the cluster nodes,
+	return fmt.Sprintf("e2e-ports-%s-%s", g.cluster, g.locationRaw), nil
```
Comment (marseel) [inline kubetest/gke.go:779]: Can you remove error in return too?

Comment (Argh4k) [reply kubetest/gke.go]: done

**Comment by marseel** [comment]
/lgtm
/assign @wojtek-t 
as I have no power here :)

**Pull request merged**